.github/scripts/build.mjs

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env zx
+
+import path from 'path'
+
+$.verbose = false
+
+let input = path.join(__dirname, '../../index.mjs')
+let output = path.join(path.dirname(input), 'index.cjs')
+
+console.log('Building')
+console.log('   input:', input)
+console.log('  output:', output)
+
+let i = 0,
+  spin = () => process.stdout.write(`  ${[...'⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏'][i++ % 10]}\r`),
+  stop = (id => () => clearInterval(id))(setInterval(spin, 100))
+
+let {stdout: code} = await $`rollup --format cjs --plugin commonjs,node-resolve ${input}`
```

Comment:
Hmm... Can rollup handle esm from cjs imports? Does it attach smth like `std/esm` to load `globby`?
If not, here's an ugly workaround example: https://github.com/qiwi/semantic-release-toolkit/blob/master/packages/plugin-creator/src/main/exports/es5.cjs

Reply:
Globby just became bundled into index.cjs with all deps.

Reply:
Nice.